rules/windows/credential_access_spn_attribute_modified.toml

```diff
@@ -77,11 +77,11 @@ DS Access >
 Audit Directory Service Changes (Success,Failure)
 ```
 
-The above policy does not cover User objects, so we need to set up an AuditRule using https://github.com/OTRF/Set-AuditRule.
+The above policy does not cover User objects, so set up an AuditRule using https://github.com/OTRF/Set-AuditRule.
```

Comment:
```suggestion
The above policy does not cover User objects, so configure an AuditRule using https://github.com/OTRF/Set-AuditRule.
```

Reply:
Thanks for recommendation. These are only meant to mimic what the security docs are for 8.2 based on the review from the Security Docs team.